codec/reflectcodec/struct_fielder.go

```diff
@@ -18,13 +18,18 @@ const (
 
 	// TagValue is the value the tag must have to be serialized.
 	TagValue = "true"
+
+	// TagValue is the value the tag must have to be serialized, this variant
+	// includes Omit Empty
+	TagWithNullableValue = "true,nullable"
```

Comment:
Should we also support `nullable,true`?

Reply:
`json` doesn't support arbitrary ordering of fields like this... so I think it's fine to require users to be explicit